src/lib/rules/html-checker/html-checker.ts

```diff
@@ -78,7 +78,7 @@ const rule: IRuleBuilder = {
                when testing the rule and `import` doesn't work here. */
             const htmlChecker = require('html-validator');
 
-            scanOptions.data = response.body.content;
+            scanOptions.data = response.body.rawContent.toString();
```

Comment:
It is related to edge15. IIRC, rawContent in edge15 is in base64 and a string in chrome